source/uwp/AdaptiveCardsObjectModel/AdaptiveCardsObjectModel.vcxproj

```diff
@@ -118,7 +118,7 @@
   </ItemDefinitionGroup>
   <ItemDefinitionGroup>
     <ClCompile>
-      <PreprocessorDefinitions>_WINRT_DLL;_SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>_WINRT_DLL;_SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING;_DISABLE_CONSTEXPR_MUTEX_CONSTRUCTOR%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Should there be a ";" between `_DISABLE_CONSTEXPR_MUTEX_CONSTRUCTOR` and `%(PreprocessorDefinitions)`

Reply:
Same for other file.